userspace/libsinsp/sinsp.h

```diff
@@ -1118,7 +1128,7 @@ VISIBILITY_PRIVATE
 	sinsp_filter* m_filter;
 	sinsp_evttype_filter *m_evttype_filter;
 	std::string m_filterstring;
-
+	set<uint32_t> m_syscalls_of_interest;
```

Comment:
unordered_set? or do we need the ordering here? could use a comment if so

Reply:
Great catch, thanks!